core/src/main/scala/org/broadinstitute/workbench/hamm/dao/GooglePriceListDAO.scala

```diff
@@ -10,24 +10,22 @@ import org.http4s.client.Client
 
 
 
-class GooglePriceListDAO[F[_]: Sync](httpClient: Client[F], uri: Uri) {
+class GooglePriceListDAO(httpClient: Client[IO], uri: Uri) {
 
-  def getGcpPriceList(): F[GooglePriceList] = {
+  def getGcpPriceList(): GooglePriceList = {
```

Comment:
What's the reason removing `F` and making this a blocking IO? Are you planning to rewrapping this in a `IO` at call site?